ang/civicase/activity/feed/directives/activity-month-nav.directive.js

```diff
@@ -109,9 +110,13 @@
      * @param {Object} params
      * @return {Array}
      */
-    function getAPICalls (overdueFirst, params) {
+    function getAPICalls (overdueFirst, params, isMyActivitiesFilter) {
       var apiCalls;
 
+      if (isMyActivitiesFilter) {
+        params.isMyActivitiesFilter = true;
+      }
```

Comment:
if the only purpose of `isMyActivitiesFilter` is to set `params.isMyActivitiesFilter = true` why can't this be set by the the caller to `getAPICalls` ? In that way we also save ourselves from one extra parameter.